formula-android/src/main/java/com/instacart/formula/integration/FragmentFlowRenderView.kt

```diff
@@ -38,13 +38,18 @@ internal class FragmentFlowRenderView(
 ) : RenderView<FragmentFlowState> {
 
     private var fragmentState: FragmentFlowState? = null
-    private val awaitingInitialization: MutableMap<FragmentKey, FormulaFragment> = mutableMapOf()
```

Comment:
Replacing this ugly logic with `FeatureProvider` and `FormulaFragmentViewFactory`